src/interaction/inspector.jl

```diff
@@ -30,28 +35,29 @@ function bbox2string(bbox::Rect2)
          x: (%0.3f, %0.3f)
          y: (%0.3f, %0.3f)
         """,
-        p0[1], p1[1], p0[2], p1[2]
+        p0[1],
+        p1[1],
+        p0[2],
+        p1[2]
     )
 end
 
 color2text(c::AbstractFloat) = @sprintf("%0.3f", c)
 color2text(c::Symbol) = string(c)
 color2text(c) = color2text(to_color(c))
-function color2text(c::RGBAf)
+color2text(c::RGBAf) =
     if c.alpha == 1.0
         @sprintf("RGB(%0.2f, %0.2f, %0.2f)", c.r, c.g, c.b)
     else
         @sprintf("RGBA(%0.2f, %0.2f, %0.2f, %0.2f)", c.r, c.g, c.b, c.alpha)
     end
-end
```

Comment:
I don't like the free floating if-block. Imo the short form is nice when it's a one-liner but as soon as it's not I think the `function` Syntax becomes more readable. I know there are a lot of these in Makie so maybe I'm on my own with that opinion...

Reply:
That's a `JuliaFormatter` bug I think, I'll fix it.